frontend/src/utils/storageDiagnostics.ts

```
@@ -0,0 +1,348 @@
+  if (diagnostics.browserInfo.storageQuota) {
+    console.log('Browser Storage Quota:', {
+      usage: formatBytes(diagnostics.browserInfo.storageQuota.usage || 0),
+      quota: formatBytes(diagnostics.browserInfo.storageQuota.quota || 0),
+      percentUsed: diagnostics.browserInfo.storageQuota.quota
+        ? (
+            ((diagnostics.browserInfo.storageQuota.usage || 0) /
+              diagnostics.browserInfo.storageQuota.quota) *
+            100
+          ).toFixed(2) + '%'
+        : 'N/A',
+    });
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)